docker-compose.yml

```diff
@@ -6,14 +6,10 @@ services:
     - "5000:8081"
     volumes:
     - ./wharf-docker-compose/traefik:/etc/traefik # Traefik configuration file
-## Uncomment to also host web via docker-compose
-#  web:
-#    build: wharf-web
+  web:
```

Comment:
Do we want to enable this by default now?

Remember to update the docs then that says that it's not enabled by default:

https://iver-wharf.github.io/#/development/getting-started?id=accessing-locally

![image](https://user-images.githubusercontent.com/22037089/120207855-3432e680-c22d-11eb-859e-968fedb35a5d.png)

Reply:
Oh I guess this is handled by `docker-compose.build.yml`

Reply:
Yep, have created PR to update docs: https://github.com/iver-wharf/iver-wharf.github.io/pull/55